### PR TITLE
Openstack Heat Ansible Role

### PR DESCRIPTION
**Pull request by saptaktakalkar**
JIRA: [AN-35](https://midobugs.atlassian.net/browse/AN-35)

### CODE REVIEW

**Comment by saptaktakalkar** [comment]
@abelboldu Role is not complete yet. Please review work done so far.

File: tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+
+- include: facts.yml
+
+- name: "openstack_heat.tasks.main: Install python openstackclient"
+  apt:
+    name: python-openstackclient
+    state: present
```
Comment (abelboldu) [inline tasks/main.yml]: This should go to packages_debian.yml

File: tests/.vagrant/machines/ubuntu/libvirt/action_provision [hunk 2]
```diff
@@ -0,0 +1 @@
+1.5:a653a8bb-107e-4a1c-85e0-94e958b85d09
```
Comment (abelboldu) [inline tests/.vagrant/machines/ubuntu/libvirt/action_provision]: all those .vagrant files should be removed from the commit.
This makes me think that we should add `.vagrant` to the gitignore

**Comment by saptaktakalkar** [comment]
@abelboldu / @ryo-tagami  This is working now. Please review.

File: tasks/configuration.yml [hunk 3]
```diff
@@ -0,0 +1,93 @@
+---
+
+- name: "openstack_heat.tasks.configuration: Configure heat api"
+  ini_file:
+    dest: "/etc/heat/heat.conf"
+    section: "{{ item.section }}"
+    option: "{{ item.option }}"
+    value: "{{ item.value }}"
+  with_items:
+    - section:  database
```
Comment (abelboldu) [inline tasks/configuration.yml]: This alignment is good looking but does not comply with style and yamlint, we should get rid of it

**Pull request merged**